pkg/webui/console/components/api-key-form/edit.js

```diff
@@ -156,6 +160,7 @@ class EditForm extends React.Component {
       id: apiKey.id,
       name: apiKey.name,
       rights: apiKey.rights,
+      expires_at: toInputDate(new Date(apiKey.expires_at)),
```

Comment:
```suggestion
      expires_at: toInputDate(new Date(apiKey.expires_at)),
```

Please use decode/encode for this as well to marhal between the format that the backend needs and the format that the date input needs.